pkg/azuredisk/azuredisk_v2.go

```diff
@@ -57,6 +62,7 @@ var controllerLeaseDurationInSec = flag.Int("lease-duration-in-sec", 30, "Freque
 var controllerLeaseRenewDeadlineInSec = flag.Int("lease-renew-deadline-in-sec", 24, "Frequency in seconds at which node driver sends heartbeat.")
 var controllerLeaseRetryPeriodInSec = flag.Int("lease-retry-period-in-sec", 30, "Frequency in seconds at which node driver sends heartbeat.")
 var partitionLabel = "azdrivernodes.disk.csi.azure.com/partition"
+var scheme = runtime.NewScheme()
```

Comment:
It doesn't appear that this needs to be a global variable. It is only allocated and used in `StartControllersAndDieOnExit`.

Reply:
Done :)